src/components/namestable/nametable-section.component.tsx

```diff
@@ -1,11 +1,56 @@
-import React from "react";
+import React, { useContext, useEffect, useMemo, useState } from "react";
 import { DataRow } from "../../logic/schedule-logic/data-row";
+import { WorkerInfoComponent } from "./worker-info.component";
+import { Drawer } from "@material-ui/core";
+import { WorkerType } from "../../common-models/worker-info.model";
+import { Sections } from "../../logic/providers/schedule-provider.model";
+import { ScheduleLogicContext } from "../schedule-page/table/schedule/use-schedule-state";
+import { ShiftsInfoLogic } from "../../logic/schedule-logic/shifts-info.logic";
 
 export interface NameTableCellOptions {
   dataRow: DataRow[];
+  workerType: WorkerType;
 }
 
-export function NameTableSection({ dataRow }: NameTableCellOptions): JSX.Element {
+export interface WorkerInfo {
+  name?: string;
+  workerType?: WorkerType;
+  requiredHours?: number;
+  actualHours?: number;
+  overtime?: number;
+  sumOfHours?: number;
+}
+
+export interface ToggleOpenState {
+  open: boolean;
+  name: string;
+}
+
+const workerInfo: WorkerInfo = { name: "" };
+
+export function NameTableSection({ dataRow, workerType }: NameTableCellOptions): JSX.Element {
```

Comment:
1. W aplikacji sekcji - to komponenty, które zawierają komórki. Natomiast ten `drawer` nie jest w żaden sposób powiązany z sekcjami. Więc moim zdaniem lepiej nazwać go `WorkerInfoDrawer`. Albo coś w tym stylu
2. Dodaj proszę interfejs który będzie przyznaczony dla tego drawera. `NameTableCellOptions` nie warto używać, bo on nie jest powiązany z drawerem w żaden sposób: `drawer` interesuje tylko imię i typ pracownika. A skąd on je dostanie nie ma znaczenia. Wiem, że `NameTableCellOptoins` teraz zawierają tylko to, ale nic nie przeszkadza temu, żeby w przyszłości te opcji się zmieniły w jakiś sposób, który będzie bardziej przenaczony dla komórki (np. może pojawić się pole `color`)
I wtedy my będziemy niepotrzebnie go przekazywać do tego drawera